cmd/util/ledger/reporters/account_reporter.go

```diff
@@ -151,7 +152,7 @@ func NewBalanceReporter(chain flow.Chain, view state.View) *balanceProcessor {
 		accounts: accounts,
 		st:       st,
 		prog:     prog,
-		intf:     fvm.NewScriptEnvironment(ctx, vm, sth, prog),
+		intf:     fvm.NewScriptEnvironment(context.Background(), ctx, vm, sth, prog),
```

Comment:
its fine to use background context here, since the scope of collecting account balance is anyway limited.